.github/workflows/publish.yaml

```
@@ -33,3 +71,18 @@ jobs:
         run: flutter pub publish --dry-run
       - name: Publish Package
         run: flutter pub publish -f
+        if: inputs.draft != true
```

for better readability we should swap the `if` to be before `run` 
```suggestion
        if: inputs.draft != true
        run: flutter pub publish -f
```